src/sentry/services/hybrid_cloud/app/impl.py

```diff
@@ -163,6 +165,30 @@ def get_related_sentry_app_components(
             ).items()
         }
 
+    def get_component_context(
```

Comment:
nit: Shouldn't this method name be plural if we're returning a list of component contexts?